src/deepsparse/transformers/pipelines/text_generation.py

```diff
@@ -711,3 +712,59 @@ def causal_mask_input_present(model_path: str) -> bool:
             inp.name == "causal_mask"
             for inp in onnx.load(model_path, load_external_data=False).graph.input
         )
+
+    def _reset_engines_cache(self):
```

Comment:
where is this being used?

Reply:
This will vanish once @rahul-tuli rebases to the current main

Reply:
Ooh good catch; I think I mistakenly added this during conflict resolution. Removing!